internal/executor/terminalwrapper/terminalwrapper.go

```diff
@@ -99,7 +98,7 @@ func New(
 			}),
 			retry.Context(ctx),
 			retry.Delay(5*time.Second), retry.MaxDelay(5*time.Second),
-			retry.Attempts(math.MaxUint32), retry.LastErrorOnly(true),
+			retry.Attempts(0), retry.LastErrorOnly(true),
```

Comment:
Is zero now infinity?

Reply:
Yes, see https://github.com/avast/retry-go/issues/43.

Reply:
Perfect! Just double checked